mix.exs

```diff
@@ -53,8 +53,8 @@ defmodule GuardianTrackable.Mixfile do
   defp deps do
     [
       {:guardian, "~> 1.0"},
-      {:ecto, "~> 3.0"},
-      {:ecto_sql, "~> 3.0.0"},
+      {:ecto, "~> 3.1"},
+      {:ecto_sql, "~> 3.1"},
```

Comment:
I'd like to support Ecto 3.0 *and* Ecto 3.1. Can we just say `~> 3.0` for both packages?

```elixir
{:ecto, "~> 3.0"},
{:ecto_sql, "~> 3.0"},
```

Reply:
Just pushed 6d4a5bc for that.